ultradns-rest/src/test/java/denominator/ultradns/MockUltraDNSRestServer.java

```diff
@@ -0,0 +1,96 @@
+package denominator.ultradns;
```

Comment:
We are working on the code coverage to make it greater than 80%. Once we deliver all the unit tests,  this class will be surely covered & you can see the same in jacoco reports at that time.